src/expressions/operator-expr.md

```diff
@@ -36,6 +53,11 @@ assert_eq!(y, 20);
 
 ## Borrow operators
 
+> **<sup>Syntax</sup>**  
+> _BorrowExpression_ :   
+> &nbsp;&nbsp; &nbsp;&nbsp; (`&`|`&&`) [_Expression_]  
```

Comment:
Why the `&&` variant here?

Reply:
Because the lexer will give us `&&` when it finds two contiguous ampersands, since `&&` is a valid symbol. The same problem occurs with `<<` and `>>` in generics/paths, and `||` in closures. Example:

```rust
let & & mut a = & & mut 10;
let &&  mut a = &&  mut 10;
```

I'm currently very busy, but I'll put some explanation in the next days, since it isn't really obvious why `&&` is accepted.

Reply:
Done